### PR TITLE
Turn off accessibility tests for not accessible components and skip rest errors

### PR DESCRIPTION
**Pull request by iBat**
1. Skipped the following tests:
- for Charts(all viz), FileManager, Gantt, Diagram, Scheduler, PivotGrid, Accordion
- all contrast issues
- reset specific errors (moved for planning, should be fixed in the nearest time).
2. We made accessibility task to be failed if some unexpected report is found.
After merge of this pull request we'll make this job required to be successfully passed.
3. Fixed issues with non-unique id in Grid's demo.

### CODE REVIEW

File: testing/common.test.js [hunk 1]
```diff
@@ -79,6 +120,16 @@ const execTestCafeCode = (t, code) => {
 
   const getDemoPaths = (platform) => glob.sync('JSDemos/Demos/*/*')
     .map((path) => join(path, platform));
+  const ACCESSIBILITY_UNSUPPORTED_COMPONENTS = [
+    'Accordion',
+    'Charts',
+    'Diagram',
+    'FileManager',
+    'Gantt',
+    // Grid's
```
Comment (EugeniyKiyashko) [inline testing/common.test.js]: Why do we need this comment?

File: testing/common.test.js [hunk 2]
```diff
@@ -54,6 +54,47 @@ const execTestCafeCode = (t, code) => {
   return testCafeFunction(t);
 };
 
+const SKIP_ACCESSIBILITY_TESTS = ['TabPanel-Overview'];
+const COMMON_SKIP_RULES = ['color-contrast'];
+const getTestSpecificSkipRules = (testName) => {
+  switch (testName) {
+  case 'Calendar-MultipleSelection':
```
Comment (ksercs) [inline testing/common.test.js]: Let's add an indent for the rows with`case`?
```javascript
switch (testName) {
    case 'Calendar-MultipleSelection':
        return ['empty-table-header']; 
```

File: testing/common.test.js [hunk 3]
```diff
@@ -130,7 +184,19 @@ const execTestCafeCode = (t, code) => {
         }
 
         if (process.env.STRATEGY === 'accessibility') {
-          const { error, results } = await axeCheck(t, '.demo-container');
+          if (SKIP_ACCESSIBILITY_TESTS.indexOf(testName) > -1) {
+            return;
+          }
+
+          const testSpecificSkipRules = getTestSpecificSkipRules(testName);
```
Comment (ksercs) [inline testing/common.test.js]:
```suggestion
          const specificSkipRules = getTestSpecificSkipRules(testName);
```

**Review by ksercs — APPROVED** [review]

File: testing/common.test.js [hunk 4]
```diff
@@ -54,6 +54,47 @@ const execTestCafeCode = (t, code) => {
   return testCafeFunction(t);
 };
 
+const SKIP_ACCESSIBILITY_TESTS = ['TabPanel-Overview'];
```
Comment (ksercs) [inline testing/common.test.js:57]: This exception and cases below will be processed separetelly - we'll remove this skips after fixes.

**Review by alexslavr — APPROVED** [review]

**Review by EugeniyKiyashko — APPROVED** [review]

**Review by pomahtri — APPROVED** [review]

**Pull request merged**